src/webrtc/call.ts

```diff
@@ -1354,32 +1260,31 @@ export class MatrixCall extends EventEmitter {
             logger.warn(`Streamless ${ev.track.kind} found: ignoring.`);
             return;
         }
+
+        let remoteStream = this.feeds.find((feed) => {return !feed.isLocal()})?.stream;
```

Comment:
I think this would be less confusing if this were just `oldRemoteStream` and then not reassigned later to be the new stream.